### ISSUE TITLE
Facilities Sprint Plan 87

### ISSUE THREAD

**Issue by kmariepat-cityfriends** [department-of-veterans-affairs/va.gov-cms#14037]
##### VBA Modernization
`All`
- Review and Incorporate Feedback from sitewide content IA team #14123 🎯

##### Facility Locator 
- Prep for Facility Locator Design Intent #14011 🎯
- User Flow Diagram for Facility Locator #14005

##### Mental Health Numbers
- Inconsistent displays for phone numbers within VAMC health service accordions #13705 🎯
- Meeting to align on approaches determined in Spike will occur this sprint, in order to refine next steps into cards for the following sprint (Sprint 88)

##### Full Width Banners
- VHA DM to confirm we can publish KBs on full width banners 
- VAMC editor are re-using VAMC System Banner Alert with Situation Updates #8141 🎯

---

##### DEV
- Force queue of VetCenter Facilities does not include all Outstations, mobiles or CAPS #13768
- Consider auto archiving VC Outstations and Mobiles when removed from source #14032
- Can't flag/save new VAMC facilities for editor contacted #13842

##### Accessibility Initiated
- VAMC Directions links SHOULD include sr-only text to indicate specific location #12085 
- Secondary nav is not programmatically associated to "In this section" button #12063

##### Help Desk Initiated 
- Will pull in flagged facility work as it becomes unblocked by editors

**Comment by jilladams** [comment]
@kmariepat-cityfriends FYI new ticket from Ryan with follow up from the flipper he audited for removal in last sprint: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14112.  Not critical, just ready to refine and could be pulled in if our one blocker from Platform team (identifying a flipper that could be used for a test) is provided in time.

**Comment by kmariepat-cityfriends** [comment]
Check in with Jordan about VBA design, after confirming recent changes are in, ping on Review and Incorporate Feedback from sitewide content IA team #14123
